Content.Server/Entry/EntryPoint.cs

```diff
@@ -114,10 +126,20 @@ public override void Init()
 
                 _voteManager.Initialize();
                 _updateManager.Initialize();
+
                 _playTimeTracking.Initialize();
                 _watchlistWebhookManager.Initialize();
+                _gitManager.Initialize(_githubWorker.Writer);
                 IoCManager.Resolve<JobWhitelistManager>().Initialize();
                 IoCManager.Resolve<PlayerRateLimitManager>().Initialize();
+
+                Task.Factory.StartNew(
+                    async () =>
+                    {
+                        await _githubWorker.HandleQueue();
+                    },
+                    TaskCreationOptions.LongRunning
+                );
```

Comment:
Again, this kind of initialization logic should be somewhere else.

Reply:
Also, just use `Task.Run()` for this. `StartNew` isn't aware of the task's own asyncness, so `TaskCreationOptions.LongRunning` will likely end up creating a new OS thread entirely which is a waste.